gemrb/core/Interface.cpp

```diff
@@ -880,6 +880,8 @@ int Interface::Init(const InterfaceConfig* cfg)
 	CONFIG_PATH("GamePortraitsPath", config.GamePortraitsPath, "portraits");
 	CONFIG_PATH("GameScriptsPath", config.GameScriptsPath, "scripts");
 	CONFIG_PATH("GameSoundsPath", config.GameSoundsPath, "sounds");
+	CONFIG_PATH("GameLanguagePath", config.GameLanguagePath, "lang/en_US");
+	CONFIG_PATH("GameMoviesPath", config.GameMoviesPath, "movies");
```

Comment:
TODO: document in config samples and man page.

Reply:
Given that we want to do it like this for now? I was just not sure if another EE or distribution requires a more comprehensive approach or something.

Reply:
The EEs are now all on the same engine, except for PST:EE, so there's not as much diversity as you think. We don't need to support older EE versions, but in this case I don't think it even matters, the layout hasn't changed from the start AFAIK.